web_source/cities/north_america/usa/denver.md

```diff
@@ -4,12 +4,14 @@ Country: USA (N. America)
 
 ##### Also known as:
 
-FIXME
+The Mile High City, Queen City of the Plains or Wall Street of the West.
 
 ### What we would like to see there
 
-FIXME
+Denver Museum of Nature & Science
+
+![Dinosaur skeletons](https://assets.simpleviewinc.com/simpleview/image/upload/c_limit,q_75,w_1200/v1/crm/denver/846_20081208_7768_MuseumofNatureandScience_b6931da7-c1ac-338c-d601817ba44cabbc.jpg)
```

Comment:
That's good, but I think we need more dinosaurs.

Reply:
Ok sounds good